### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by cra0zy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,63 +4,65 @@
...
-* [Windows 10 UWP](http://dev.windows.com)
-* [OUYA](https://www.ouya.tv/), an Android-based gaming console
+ * Forum can be found at: [community.monogame.net](http://community.monogame.net/)
+ * Chat with the community at:&nbsp; [![Join the chat at https://gitter.im/mono/MonoGame](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/mono/MonoGame?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
+ * Documentation is available at: [www.monogame.net/documentation/](http://www.monogame.net/documentation/)
+ * [Download Binaries](http://www.monogame.net/downloads/).
+ * Follow [@MonoGameTeam on Twitter](https://twitter.com/monogameteam).
 
-## Quick Start
+### Supported Platforms
```
Comment (tomspilman) [inline README.md]: I think we have to still specify Windows 10 and Windows 8 separately.  Really anything that protobuild can generate should likely be in this list.

File: README.md [hunk 2]
```diff
@@ -4,63 +4,65 @@
...
-* Download the latest [source code](https://github.com/mono/MonoGame/archive/develop.zip).
-* Fork and clone the repo: `https://github.com/mono/MonoGame.git`.
+Mobile:
+ * Android
+ * iOS
+ * Windows Phone
 
-### Solutions & Projects
+Console:
+ * Playstation 4
```
Comment (tomspilman) [inline README.md]: It should be "PlayStation 4" also we can add "PlayStation Vita" here too.

For those platforms we need to add a note like "contact your Sony account manager for access".

File: README.md (hunk 1 above)
Comment (cra0zy) [inline README.md]: Better now?

File: README.md [hunk 3]
```diff
@@ -4,63 +4,65 @@
...
-The solution and project files are generated by [Protobuild](https://github.com/hach-que/Protobuild) when you double-click `Protobuild.exe` on Windows or run `mono Protobuild.exe` under Mac OS or Linux.  To modify the projects you must edit the .definition file in the `Build/Projects/` folder and re-execute Protobuild.
+### Source Code
 
-For more information and advanced usage, please refer to the [Protobuild documentation](https://protobuild.readthedocs.org/).
+ * Clone the source: `git clone https://github.com/mono/MonoGame.git`
+ * Run Protobuild.exe to generate project files and solutions.
+   * If on Linux or Mac, run it with mono: `mono Protobuild.exe`
 
-### Samples
+### Building
```
Comment (tomspilman) [inline README.md]: Maybe this section should be combined with "Source Code"?

**Comment by cra0zy** [comment]
@dellis1972 PING... few days late...

**Pull request merged**